jekyll/_cci2/openid-connect-tokens.adoc

```diff
@@ -112,7 +114,7 @@ You will need to allow your AWS account to trust CircleCI's OpenID Connect token
 
 Visit the https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_create_oidc.html[Creating OpenID Connect (OIDC) identity providers] page of the AWS docs and follow the instructions. You will need your CircleCI Organization ID, which you can find by navigating to **Organization Settings > Overview** on the https://app.circleci.com/[CircleCI web app]. Copy your Organization ID for the next step.
 
-When asked for the **Provider URL**, enter: `\https://oidc.circleci.com/org/ORGANIZATION_ID`, where `ORGANIZATION_ID` is the ID of your CircleCI organization. Provide the same CircleCI Organization ID for the **Audience**.
+When asked for the **Provider URL**, enter: `\https://oidc.circleci.com/org/<organization_id>`, where `organization_id` is the ID of your CircleCI organization. Provide the same CircleCI Organization ID for the **Audience**.
```

Comment:
@rosieyohannan When we indicate that part of a string needs to be replaced with a value, should we include the `< >` , e.g. "...where `<organization_id>` is the ID of your CircleCI org"?

Reply:
I think the way you have it there without `< >` in the text is right 👍